src/javascript/_common/base/client_base.js

```diff
@@ -135,6 +135,19 @@ const ClientBase = (() => {
         return only_real_loginids.every(loginid => get('currency', loginid) && !isCryptocurrency(get('currency', loginid)));
     };
 
+    const isWalletsAccount = (loginid) => {
+        if (typeof loginid === 'undefined') {
```

Comment:
undefined here is a string, is it correct?

Reply:
yes